### PR TITLE
Feature/frontend improvements

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

**Review by Thykof — CHANGES_REQUESTED** [review]
- transform from a create-react-app to a vite app and tailwindcss https://tailwindcss.com/docs/guides/vite
- add linter and formatter to uniformise with other project and avoid error
    - our standards https://github.com/massalabs/prettier-config-as/ and https://github.com/massalabs/eslint-config
- add CI
    - get inspiration from https://github.com/massalabs/station-massa-wallet/blob/main/.github/workflows/lint.yml#L33
- use the ui kit https://github.com/massalabs/Ui-Kit/
- use tailwindcss, remove manual CSS in the code, it’s harder to maintain because it’s the only project like this, tailwindcss allow to code faster and is well integrated with Lucas’s works on the designs
- add internationalisation of the texts shown to the user
    - we also have a small lib in our project to anticipate internationalisation: https://github.com/massalabs/station-massa-wallet/tree/main/web-frontend/src/i18n
    - we decided to create a minimalist library and put all text in a json file with only one language (en_US) because it doesn’t take more time to do that and will save days when we will want to add a new language
- Connect Massa Wallet: we can reuse the whole work on the bridge (many weeks of development)

Those standards improve the quality of the code, accelerate development and code review, avoid errors. Projects always have bugs that we need to fix and users will use this project for at leat 6 months (vesting period). So we must anticipate bug resolution and new feature to be developed by new developers. By using the same tech stack as in other projects, we take benefit of team skill that has been developed since 2 years.

- estimation 1 day to set up vite app and tailwindcss, linter, formatter, CI
- estimation 2 days to move texts to i18n (internationalisation) and use the ui kit

File: coin-vester-front/src/pages/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+        throw new Error("No providers available");
+      }
+
+      const walletProvider = allProviders.find(
+        (provider) => provider.name() === walletProviderName
+      );
+
+      if (!walletProvider) {
+        setError(
+          `Wallet ${walletProviderName} not found.\nPlease make sure it is installed and running.`
```
[the remark this answers is not in the document]
Comment (thomas-senechal) [reply coin-vester-front/src/pages/HomePage.tsx:52]: It renders correctly while `<br/>` renders as `<br/>`

[the remark this answers is not in the document]
Comment (thomas-senechal) [reply coin-vester-front/src/pages/HomePage.tsx:52]: In fact, both are not working
I removed `\n`s

**Comment by thomas-senechal** [comment]
We will merge this PR to use it as the base of our changes which will include: 
- moving to vite
- add linter
- add ci
- add UI Kit and Tailwind CSS
- remake Home and Send pages with UI Kit and Tailwind CSS
- use bridge wallet handler to handle wallets

**Pull request merged**